dagstore_control.go

```diff
@@ -52,11 +62,12 @@ func (d *DAGStore) control() {
 			return
 		}
 
-		if gc != nil {
+		// Disable manaual GC if client wants automated LRU GC.
+		/*if gc != nil {
 			// this was a GC request.
 			d.gc(gc)
 			continue
-		}
+		}*/
```

Comment:
Automated GC should not preclude manual GC requests from happening. Any garbage collector there (e.g. JVM, Go, etc.) will honour manual GC runs.